samples/gen-go-wag-patch/client/client.go

```diff
@@ -0,0 +1,169 @@
+package client
```

Comment:
I think we forgot to commit these in in a past PR - when I ran the tests they got generated.